templates/patterns/icons_with_text/pattern-icons-with-text.html.twig

```diff
@@ -0,0 +1,10 @@
+{#
+/**
+ * @file
+ * A list of icons with a text.
+ */
+#}
+{% include '@oe_theme/compositions/ec-component-icons-with-text/icons-with-text.html.twig' with {
+  'items': items|default([]),
```

Comment:
Why do you provide default if you already provide this default in the included twig?

Reply:
I agree. removed.